runtime/dancebox/src/xcm_config.rs

```diff
@@ -477,7 +479,25 @@ parameter_types! {
     pub const XcmBuyExecutionDotRococo: u128 = XCM_BUY_EXECUTION_COST_ROCOCO;
```

Comment:
this is not being used anywhere AFAICT, is it just for testing=?

Reply:
Yes. But, I think we should have some limit imposed in proper runtime there.